build.sbt

```diff
@@ -35,7 +35,7 @@ libraryDependencies ++= Seq(
   "org.slf4j" % "log4j-over-slf4j" % "1.7.12",
   "com.adrianhurt" %% "play-bootstrap3" % "0.4.5-P24",
   "org.clapper" %% "grizzled-slf4j" % "1.0.2",
-  "org.apache.kafka" %% "kafka" % "0.8.2.2" exclude("log4j","log4j") exclude("org.slf4j", "slf4j-log4j12") force(),
+  "org.apache.kafka" %% "kafka" % "0.9.0.0" exclude("log4j","log4j") exclude("org.slf4j", "slf4j-log4j12") force(),
```

Comment:
Hi there, Kafka 0.9.0.1 was released a couple of days ago. It fixes numerous issues, I think this PR should rely on the fix version.